### ISSUE TITLE
[FEATURE] Use a topic specific example in the description for `meta`

### ISSUE THREAD

**Issue by fabianegli** [nf-core/modules#4744]
### Is your feature request related to a problem? Please describe

The example in the sample description is sequencing specific and this irritates me when I see it in a module which is not specific to sequencing analyses. e.g.

https://github.com/nf-core/modules/blob/4196b1b2e7ce265892f3979eabf7a9ddc030702f/modules/nf-core/openms/idfilter/meta.yml#L37

### Describe the solution you'd like

It would be great if the examples, if they are given at all, were topic and module specific.

**Comment by mashehu** [comment]
Yeah, I also don't like it. Any idea what best to do about it? I was thinking of throwing a lintinn warning I'd it is the same as in the template.

**Comment by famosab** [comment]
@fabianegli Maybe examples can also added to the docs and the template can be kept clean?

**Comment by vagkaratzas** [comment]
This has been implemented and the template now outputs `[ id:'sample1' ]` on module creation.
